password_cracker.md

```diff
@@ -15,6 +15,14 @@ wikibook:
   - "Synchronization, Part 5: Condition Variables"
   - "Synchronization, Part 6: Implementing a barrier"
 ---
+## Backstory
+After finally being taken off of PIP due to your outstanding malloc performance you can finally relax a little. So take a Pomelo ElCroix out of the minifridge and relax!

```

Comment:
```suggestion
After finally being taken off of PIP due to your outstanding malloc performance, you can finally relax a little. So you take a Pomelo ElCroix out of the mini-fridge and relax!
```

Reply:
Tonally I felt it was better to exclude the "you" so it came across as more of an encouragement and less as a narration

Reply:
Sounds good!